app/products/playbooks/screens/playbook_run/checklist/checklist_item/checklist_item.tsx

```diff
@@ -137,6 +138,24 @@ const ChecklistItem = ({
         setIsChecking(false);
     }, [isChecking, serverUrl, playbookRunId, item.id, checklistNumber, itemNumber, checked]);
 
+    const toggleSkipped = useCallback(async () => {
+        if (isChecking) {
+            return;
+        }
+        setIsChecking(true);
+        let res;
+        if (skipped) {
+            res = await restoreChecklistItem(serverUrl, playbookRunId, item.id, checklistNumber, itemNumber);
+        } else {
+            res = await skipChecklistItem(serverUrl, playbookRunId, item.id, checklistNumber, itemNumber);
+        }
+        if (res.error) {
+            showPlaybookErrorSnackbar();
+            logDebug('skipChecklistItem error', res.error);
```

Comment:
What is the point of this log? Looking at the action the lot is already there, no? And in any case the label should be toggleSkipped